modules/ztp-install-factory-spoke-pipeline.adoc

```diff
@@ -0,0 +1,186 @@
+        bmc_url: "<url bmc>" <8>
+        bmc_user: "user-bmc" <9>
+        bmc_pass: "user-pass" <10>
+        root_disk: sda <11>
+        storage_disk: <12>
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master1:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static: eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:11"
+        mac_int_static: "aa:ss:dd:ee:b1:11"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        root_disk: sda
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master2:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static: eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:12"
+        mac_int_static: "aa:ss:dd:ee:b1:12"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        root_disk: sda
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+    worker0: <13>
+        nic_ext_dhcp: eno4
+        nic_int_static: eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:19"
+        mac_int_static: "aa:ss:dd:ee:b1:19"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+  - spoke2-name:
+      master0:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:20"
+        mac_int_static: "aa:ss:dd:ee:b1:20"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master1:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:21"
+        mac_int_static: "aa:ss:dd:ee:b1:21"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      master2:
+        ignore_ifaces: eno1 eno2
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:22"
+        mac_int_static: "aa:ss:dd:ee:b1:22"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+        storage_disk:
+          - sdb
+          - sdc
+          - sde
+          - sdd
+      worker0:
+        nic_ext_dhcp: eno4
+        nic_int_static:  eno5
+        mac_ext_dhcp: "aa:ss:dd:ee:b0:29"
+        mac_int_static: "aa:ss:dd:ee:b1:29"
+        bmc_url: "<url bmc>"
+        bmc_user: "user-bmc"
+        bmc_pass: "user-pass"
+----
+<1> This option is configurable and sets the name of the edge cluster.
+<2> This value must match `master0`, `master1` or `master2`.
+<3> Optional: Interfaces to ignore in the host.
+<4> NIC connected to the external DHCP.
+<5> NIC connected to the internal network.
```

Comment:
The build is showing an error where there isn't a <5> callout in the code block above:

```
asciidoctor: WARNING: modules/ztp-install-factory-spoke-pipeline.adoc: line 149: no callout found for <5>
```

There is a `#` on that line that is causing the issue